code/game/jobs/job/security.dm

```diff
@@ -136,13 +136,14 @@
 
 	implants = list(/obj/item/implant/mindshield)
 
-/datum/outfit/job/detective/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+/datum/outfit/job/detective/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE, alt_title)
 	. = ..()
-	if(H.mind.role_alt_title)
-		switch(H.mind.role_alt_title)
-			if("Forensic Technician")
-				suit = /obj/item/clothing/suit/storage/det_suit/forensics/blue
-				head = null
+	if(!alt_title)
+		alt_title = H.mind && H.mind.role_alt_title ? H.mind.role_alt_title : null
+	if(alt_title && alt_title == "Forensic Technician" )
```

Comment:
There's no need to check `if(alt_title` here, `==` can't runtime from null values.